content/hardware/01.mkr/01.boards/mkr-wan-1310/tutorials/lora-sensor-data/lora-sensor-data.md

```diff
@@ -9,7 +9,7 @@ tags:
   - Environmental data
 author: 'Karl Söderby'
 libraries: 
-  - name: LoRa
+  - name: LoRa®

```

Comment:
Thats the library name, so we should sync with the library's name
```suggestion
  - name: LoRa
```

Reply:
changed